src/components/authentication/forgotPassword.js

```diff
@@ -0,0 +1,44 @@
+import { useNavigate } from "react-router";
+
+export function ForgotPassword() {
+  const navigate = useNavigate();
+  return (
+    <div className="margin-top">
+      <form className="form-box">
+        <h2 class="text-centre">Forgot Password</h2>
+        <div className="input-field">
+          <input type="email" className="input-text" id="email" required />
```

Comment:
input can be nested inside label and written like
`<label>Email:<input/></label>` 
So that the id can be avoided